idea/gear/frontend/src/features/dns/consts/sails.ts

```diff
@@ -1,86 +1,89 @@
-import { GearApi } from '@gear-js/api';
-import { TypeRegistry } from '@polkadot/types';
-import { TransactionBuilder } from 'sails-js';
-
-export type ActorId = `0x${string}`;
+/* eslint-disable @typescript-eslint/no-explicit-any */
 
-export interface ContractInfo {
-  admin: ActorId;
-  program_id: ActorId;
-  registration_time: string;
-}
+import { GearApi, Program } from '@gear-js/api';
+import { TypeRegistry } from '@polkadot/types';
+import { TransactionBuilder, ActorId } from 'sails-js';
 
-export class Program {
+export class SailsProgram {
```

Comment:
don't we want to have a meaningful name here (like DnsProgram or sth else)?

Reply:
All `dns` related data is in its related feature folder, there's no other Sails program in it, therefore I think default `SailsProgram` is sufficient and descriptive. Like, there's `dns` feature -> it has its Sails program.